src/components/tracked-controls.js

```diff
@@ -164,13 +164,18 @@ module.exports.Component = registerComponent('tracked-controls', {
     }
 
     // Apply transforms, if 6DOF and in VR.
-    if (vrDisplay) {
+    if (vrDisplay && pose.position !== null) {
```

Comment:
@dmarcos After testing in Daydream, I found that this change was necessary as well. Just wanted to ensure it makes sense to you as well.

Reply:
It makes sense, we can just make position` (vrDisplay && pose.position)`